nav2_map_server/src/costmap_filter_info/costmap_filter_info_server.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 Samsung Research Russia
```

Comment:
This needs test coverage. This should also be very simple. Just create an instance of this server and create a simple python node just to check that it got the values on the topics.

Reply:
Agree with it. How about to make a separate PR for testing update. If you let me, I would like to remain this test as a step-by-step and write a new python-based system test as you mentioned, for local and global costmaps verification.

Reply:
I'm OK delaying for the full system test, but unit / component tests need to be in place.

Reply:
Also the lack of testing here drops map_server package below 90%, which I won't merge until back up. I spent too long writing tests to get these packages up at 90% to let it slide

Reply:
My apologies, I thought that you are talking about system tests in this thread.
I've added new unit testcase for `CostmapFilterInfoServer` in https://github.com/ros-planning/navigation2/commit/ab3f5fe2e5f47906832bfee91c2b6ff8e280eb8d#diff-cf1a32f3476d8c2ae229c9b55c11a91a